platform/mikro-e/Makefile.mikro-e

```diff
@@ -15,9 +15,17 @@ CONTIKI_PLAT_DEFS += -D __USE_UART_PORT3__
 ifeq ($(USE_SERIAL_PADS),1)
   #If using RB9 and RD11 as uart pins for debug
   CONTIKI_PLAT_DEFS += -D __USE_UART_PORT2_FOR_DEBUG__
+ifeq ($(USE_SLIP),1)
+  CONTIKI_PLAT_DEFS += -D __USE_UART_PORT3__
```

Comment:
Francois's change already added this by default, we can get rid of the extra -D __USE_UART_PORT3__